client/client.cpp

```diff
@@ -654,11 +668,12 @@ void* write_poll(void* args)
     while (poll)
     {
         int num_bytes_tcp = read(socks->tcp_sock, write_buf, BUF_SZ);
-        if (num_bytes_tcp == -1)
+        if (num_bytes_tcp < 0)
         {
-            // This happens when there is a timeout, we simply keep looping, as we want do not want to block while
+            // Return value of -1 happens when there is a timeout, we simply keep looping, as we want do not want to block while
```

Comment:
Please fix comment or remove `continue`

Reply:
I think comment is correct. If there is a timeout from the read then we `simply keep looping`.  The `shut down the threads` part comes from the the `poll` variable is set to false when the thread should stop. I'll update the `while writing to the CAN-Bus` part since that should be `while reading from TCP`